3rdparty/openssl/CMakeLists.txt

```diff
@@ -98,26 +253,18 @@ ExternalProject_Add(
   SOURCE_DIR ${OPENSSL_DIR}
   CONFIGURE_COMMAND
   COMMAND ${CMAKE_COMMAND} -E copy_directory ${OPENSSL_DIR}/include/openssl
-          ${OPENSSL_INSTALL_INCLUDES}
-  COMMAND
-    ${CMAKE_COMMAND} -E copy ${OPENSSL_BUILD_INCLUDES}/openssl/opensslconf.h
-    ${OPENSSL_INSTALL_INCLUDES}/opensslconf.h
+          ${OPENSSL_INSTALL_INCLUDES} ${OPENSSL_INCLUDES_COMMANDS}
```

Comment:
I would recommend removing the very first COMMAND of OPENSSL_INCLUDES_COMMANDS, and then make this as
`COMMAND ${OPENSSL_INCLUDES_COMMANDS}`